src/Storage/src/Storage.php

```diff
@@ -36,11 +36,9 @@ final class Storage implements MutableStorageInterface
      * @var array<string, BucketInterface>
      */
     private array $buckets = [];
-    private string $default;
 
-    public function __construct(string $name = self::DEFAULT_STORAGE)
+    public function __construct(private string $default = self::DEFAULT_STORAGE)
```

Comment:
We can't rename parameters in public signatures in v3

Reply:
sure, reverted, updated the configured rule as disallow rename property https://github.com/spiral/framework/pull/1139/commits/ccd058aae09a77189f31f1b45b20203664efb925